python/lsst/pipe/base/pipelineIR.py

```diff
@@ -619,10 +619,14 @@ def process_args(argument: Union[str, dict]) -> dict:
             if self.instrument is None:
                 self.instrument = tmp_IR.instrument
             elif self.instrument != tmp_IR.instrument and tmp_IR.instrument is not None:
-                raise ValueError("Only one instrument can be declared in a pipeline or it's imports")
-            if accumulate_tasks.keys() & tmp_IR.tasks.keys():
-                raise ValueError("Task labels in the imported pipelines must "
-                                 "be unique")
+                msg = ("Only one instrument can be declared in a pipeline or its imports. "
+                       f"These instruments were found: {self.instrument}, {tmp_IR.instrument}")
+                raise ValueError(msg)
+            duplicate_labels = accumulate_tasks.keys() & tmp_IR.tasks.keys()
+            if duplicate_labels:
```

Comment:
You are free to leave it as is, but this is exactly the kind of place warlrus was designed for, variable is only used in one context:
```
if duplicate_labels := accumulate_tasks.keys() & tmp_IR.tasks.keys():
    msg = ("Task labels in the imported pipelines must be unique. "
           f"These labels appear multiple times: {duplicate_labels}")
```